### PR TITLE
Adapt UI tests to updated rsp-ui

### PR DESCRIPTION
**Pull request by odockal**
Signed-off-by: Ondrej Dockal <odockal@redhat.com>

### CODE REVIEW

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
testPR

**Comment by odockal** [comment]
@jrichter1 PR check are green, could you please check this out? Thanks

File: src/ui-test/basicServerTest.ts [hunk 1]
```diff
@@ -51,8 +49,11 @@ export function wildflyE2EBasicTest() {
                 expect(serversNames).to.include.members([serverName]);
                 const server = await serverProvider.getServer(serverName);
                 await server.start();
+                await VSBrowser.instance.driver.wait( async () => { return await serverHasState(server, ServerState.Started);}, 3000 );
```
Comment (jrichter1) [inline src/ui-test/basicServerTest.ts:52]: `driver` is already initialized as a shorthand for `VSBrowser.instance.driver`, why not use it?

Comment (odockal) [reply src/ui-test/basicServerTest.ts:52]: true, true....

**Pull request merged**